schemas/database/002_create_root_history.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE root_history
```

Comment:
this should also track root status w.r.t. chain status (i.e. is it even a mined-batch root and, in the future, is it confirmed)